.editorconfig

```diff
@@ -1,2 +1,11 @@
-[*.{kt,kts}]
-disabled_rules=no-wildcard-imports
\ No newline at end of file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+insert_final_newline = true
```

Comment:
@TacoTheDank I think most of these rules are already true by default. Kindly check for the ones that are true and have no need for setting it here.

Reply:
Actually, I don't think these are explicitly set by default unless specifically set by something such as an .editorconfig or some other tool. I've come across many an Android project without .editorconfigs (or similar things) that have had their codestyles all over the place (newlines and no newlines, tabs and spaces being used at the same time, etc)...